tools/validate/validate.cpp

```diff
@@ -19,15 +19,16 @@ constexpr size_t max_line_length = 120;
 
 class BinaryFile {
 public:
-    explicit BinaryFile(const filesystem::path& filepath) {
-        m_file = _wfopen(filepath.c_str(), L"rb");
-
+    explicit BinaryFile(const filesystem::path& filepath)
+        // clang-format off
+    : m_file(_wfopen(filepath.c_str(), L"rb")) {
+        // clang-format on

```

Comment:
Suggestion: Instead of suppressing clang-format, it's simpler to add an empty comment when you want to force a line-wrap. This consumes fewer lines, and allows clang-format to handle indentation etc.